homeassistant/components/lock/__init__.py

```diff
@@ -107,10 +107,6 @@ async def _async_unlock(entity: LockEntity, service_call: ServiceCall) -> None:
 
 async def _async_open(entity: LockEntity, service_call: ServiceCall) -> None:
     """Open the door latch."""
-    if not (entity.supported_features & LockEntityFeature.OPEN):
```

Comment:
When we call from the service from a group we need to see of an entity supports OPEN, how to do that?

Reply:
Added tests to make sure that `LockEntityFeature.OPEN` validates on a group of locks.